### ISSUE TITLE
[Writing program content form] Allow selecting multiple types of content

### ISSUE THREAD

**Issue by tinniaru3005** [layer5io/layer5#3449]
#### Description

In the writing form currently, we can just select one type of content. But someone might be interested in multiple types of content. So it can be changed so that multiple options can be selected

#### Page Link:

[Writing program](https://layer5.io/community/handbook/writing-program)

#### Expected Behavior

Allow selected multiple types of content in the writing program form

#### Screenshots

![Layer5-writing-program](https://user-images.githubusercontent.com/62856848/201408102-419a5ca9-a592-4704-8d51-be35a853f0cf.png)

#### Environment:
- Host OS: 
- Browser: 

---
<img src="https://raw.githubusercontent.com/layer5io/layer5/master/.github/assets/images/layer5/5-light-small.svg" width="16px" align="left" /><h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

**Comment by tinniaru3005** [comment]
If the changes seem fine, I would like to work on it.

**Comment by shivam-sharma7** [comment]
hey @tinniaru3005 are you working on this? if you not, so you can assign me

**Comment by tinniaru3005** [comment]
Hey I didn't start working on it as it was not assigned. Let someone confirm whether the changes are fine or not before starting to work.

**Comment by UtkarshMishra12** [comment]
@leecalcote @Nikhil-Ladha @Chadha93 ^^

**Comment by Nikhil-Ladha** [comment]
It may be needed, but not a priority as of now.
I will wait for @debo19 to be back before anyone starts working on this, as changing it to multi-select would require changes around the API calls and the integrations as well.

**Comment by UtkarshMishra12** [comment]
@Nikhil-Ladha May I close this issue 👀 ?
